query-engine/black-box-tests/tests/metrics/smoke_tests.rs

```diff
@@ -55,22 +55,54 @@ mod smoke_tests {
+            assert_eq!(metrics.matches("# TYPE prisma_pool_connections_closed_total counter").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_pool_connections_opened_total The total number of pool connections opened").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_pool_connections_opened_total counter").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_client_queries_active The number of currently active Prisma Client queries").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_client_queries_active gauge").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_client_queries_wait The number of datasource queries currently waiting for an free connection").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_client_queries_wait gauge").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_pool_connections_busy The number of pool connections currently executing datasource queries").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_pool_connections_busy gauge").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_pool_connections_idle The number of pool connections that are not busy running a query").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_pool_connections_idle gauge").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_pool_connections_open The number of pool connections currently open").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_pool_connections_open gauge").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_client_queries_duration_histogram_ms The distribution of the time Prisma Client queries took to run end to end").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_client_queries_duration_histogram_ms histogram").count(), 1);
+
+            assert_eq!(metrics.matches("# HELP prisma_client_queries_wait_histogram_ms The distribution of the time all datasource queries spent waiting for a free connection").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_client_queries_wait_histogram_ms histogram").count(), 1);
+            
+            assert_eq!(metrics.matches("# HELP prisma_datasource_queries_duration_histogram_ms The distribution of the time datasource queries took to run").count(), 1);
+            assert_eq!(metrics.matches("# TYPE prisma_datasource_queries_duration_histogram_ms histogram").count(), 1);
+            
+            // Check that exist as many metrics as being accepted
+            let accepted_metric_count = query_engine_metrics::ACCEPT_LIST.len();
+            let displayed_metric_count = metrics.matches("# TYPE").count();
+            let non_prisma_metric_count = displayed_metric_count - metrics.matches("# TYPE prisma").count();
+            
```

Comment:
OT: Maybe another assertion that we have 4 counters, 4 gauges and 3 histograms? Or is that repetitive?